schemas/aws/terraform-state-1.yml

```diff
@@ -0,0 +1,17 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /aws/account-1.yml
```

Comment:
this line needs to mention the schema itself. have a look at how other schema files are doing this

Reply:
Done, thank you for catching that!